### PR TITLE
Translations Update, please integrate

### PR DESCRIPTION
**Pull request by rgeraghty**
please merge. cc @rgeraght

### CODE REVIEW

**Review by rofe — CHANGES_REQUESTED** [review]
@rgeraghty I'd like to keep the gender of the sidekick (male) as is since people are already used to it and changing it now may be confusing.

Also, this PR accidentally reverts the localized chrome web store URLs.

**Comment by rgeraghty** [comment]
@rofe the URL's are corrected, for the mascaline change, is that for German?
Screenshot 2024-10-29 at 8 55 17 AM

File: src/extension/_locales/de/messages.json [hunk 1]
```diff
@@ -634,7 +634,7 @@
     "message": "Benutzer/in wechseln"
   },
   "v7_context_menu": {
-    "message": "Installieren Sie den neuen AEM Sidekick"
+    "message": "Den neuen AEM Sidekick schließen"
```
Comment (rofe) [inline src/extension/_locales/de/messages.json:637]:
```suggestion
    "message": "Den neuen AEM Sidekick installieren"
```

**Review by rofe — CHANGES_REQUESTED** [review]

File: src/extension/_locales/de/messages.json (hunk 1 above)
Comment (rgeraghty) [reply src/extension/_locales/de/messages.json]: @rofe translation has been updated,

**Review by rofe — APPROVED** [review]

**Pull request merged**